query_optimizer/ExecutionGenerator.cpp

```diff
@@ -128,6 +130,8 @@ using std::vector;
 
 namespace quickstep {
 namespace optimizer {
+DEFINE_int32(num_partitions, 1, "Number of partitions for the new tables");
+DEFINE_int32(partition_attr_id, 0, "The ID of the partitioning attribute in the relation");
```

Comment:
@rogersjeffreyl If I understand correctly, such info should be specified in the `CreateTable` statement, instead of global variables set during the initialization of the executable.